realm/realm-library/src/objectServer/java/io/realm/RealmFileUserStore.java

```diff
@@ -54,8 +54,20 @@ public SyncUser get() {
      * {@inheritDoc}
      */
     @Override
-    public void remove() {
-        nativeLogoutCurrentUser();
+    public SyncUser get(String identity) {
+        String userJson = nativeGetUser(identity);
+        if (userJson != null) {
+            return SyncUser.fromJson(userJson);
+        }
+        return null;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void remove(String identity) {
+        nativeLogoutUser(identity);
```

Comment:
We had a short chat and the conclusion is that this is a naming issue of `object-store`.
We would keep current code.